eawx-build-test/Tasks/CopyTaskTest.cs

```diff
@@ -3,27 +3,23 @@
 using EawXBuild.Exceptions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace EawXBuildTest.Tasks
-{
+namespace EawXBuildTest.Tasks {
```

Comment:
The formatting war has begun :P

Reply:
So namespaces and classes with braces on the same line, methods, if statements, loops on the next line?

Reply:
I'd actually have to read that up - I'm fully reliant on Rider's auto-formatter, which AFAIK follows [Microsoft's default suggestions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions).

From what I've seen from glancing over the linked docs, curly brackets always go on a new line. I don't have a preference as long as all contributors use the same fromatting just to avoid unnecessary format based changes.

Reply:
I'm using that too
Don't know why it hasn't formatted it properly

Reply:
It sometimes doesn't on save for reasons unknown and you have to trigger by hand.
I wouldn't worry too much about it.